editor/inspector/assets/fbx/model.js

```diff
@@ -669,9 +894,14 @@ exports.methods = {
         this.dispatch('change');
         this.dispatch('track', { tab: 'model', prop, value: event.target.value });
     },
+    apply() {
+        this.$.loadMask.style.display = 'flex';
+    }
 };
 
 exports.ready = function() {
+    Editor.Message.addBroadcastListener('fbx-inspector:apply', this.apply.bind(this));
```

Comment:
Need to cache a variable equal to `this.apply.bind(this)` , and use it on close period

Reply:
you are right. `bind()` wil return a new function each time